.github/workflows/validate.yml

```diff
@@ -29,3 +29,4 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: ./build.sh
+      - run: yarn test:build
```

Comment:
@AndyClifford Suggestion: Call these "integration" tests. "Build" is an overloaded term and it's not clear what it means in this context